content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/getting-started.mdx

```diff
@@ -0,0 +1,118 @@
+---
+doctype: "Chapter"
+chapterTitle: "Getting Started"
+description: "Meshery is the service mesh managemen plane which offers lifecycle, configuration, and performance management of service meshes and their workloads."
+videos: 6
+lectures: 8
+---
+
+import { ChapterStyle } from "../../../../src/components/Learn-Components/Chapters-Style/chapters.style.js";
+import LinkerdAdapter from "../../../../src/assets/images/learn-layer5/linkerd/linkerd-adapter.png";
+import InstallLinkerd from "../../../../src/assets/images/learn-layer5/linkerd/install-linkerd.png";
+
+<ChapterStyle>
+
+<h2 class="chapter-sub-heading">Deploy Linkerd</h2>
+
+Now that we have a Kubernetes cluster and Meshery, we are ready to download and deploy Linkerd resources.
+
+<h2 class="chapter-sub-heading">Steps</h2>
+
+1. [Install Linkerd](#1)
+1. [Verify install](#2)
+1. [Confirm add-ons](#3) 
+
+<h3 class="chapter-sub-heading">
+  <a name="1">Install Linkerd</a>
+</h3>
+
+Using Meshery, select `Linkerd` from the `Management` menu.
+
+<a href={LinkerdAdapter}>
+  <img src={LinkerdAdapter} />
+</a>
+
+In the Linkerd management page:
+
+1. Type `linkerd` into the namespace field.
+1. Click the (+) icon on the `Install` card and select `Latest Linkerd` to install the latest version of Linkerd.
+
+   <a href={InstallLinkerd}>
+     <img src={InstallLinkerd} align="center" />
+   </a>
+
+<h3 class="chapter-sub-heading">
```

Comment:
Move this section after the manual installation section below.

Reply:
`Verify install` section below `Manual Install` section, right?

Reply:
Yup.